makefile.common

```diff
@@ -0,0 +1,62 @@
+#common make targets shared over multiple makefiles
+
+bins: $(USEFUL_DEMOS)
+
+all_test: test tv_gen $(DEMOS)
+
+#build the doxy files (requires Doxygen, tetex and patience)
+doxygen:
+	$(MAKE) -C doc/ doxygen V=$(V)
+
+doxy:
+	$(MAKE) -C doc/ doxy V=$(V)
+
+docs:
```

Comment:
Would not it be better to have:
```
docs doc/crypt.pdf:
	$(MAKE) -C doc/ crypt.pdf V=$(V)
```
or
```
docs: doc/crypt.pdf
doc/crypt.pdf:
	$(MAKE) -C doc/ crypt.pdf V=$(V)
```
otherwise the main makefile has no idea how to make `doc/crypt.pdf` (which is used in `install`)